main.go

```diff
@@ -469,6 +469,10 @@ func main() {
 		*s3prefix = *logURL
 	}
 
+	if strings.HasPrefix(*s3bucket, "s3://") {
+		*s3bucket = strings.TrimPrefix(*s3bucket, "s3://")
+	}
```

Comment:
If the purpose of this change is to correct someone erroneously passing a URI such as `s3://<bucket-path>` or  `http://<bucket-path>` (which is what minio uses), perhaps we should try to parse this as a URI and fail if they tried to pass us a scheme?